src/Model/ProductManager.php

```diff
@@ -76,7 +76,7 @@ public function insert(array $product): int
         $statement->bindValue(`description`, $product['description'], PDO::PARAM_STR);
         $statement->bindValue(`material`, $product['matter'], PDO::PARAM_STR);
         $statement->bindValue(`color`, $product['palette'], PDO::PARAM_STR);
-        $statement->bindValue(`category_item_id`, $product['category'], PDO::PARAM_STR);
+        $statement->bindValue(`category_item_id`, $product['category'], PDO::PARAM_INT);
```

Comment:
Can you cnfirm the category key retrieved will be equal to the category_id and not the category name ?